var/spack/repos/builtin/packages/hpctoolkit/package.py

```diff
@@ -18,6 +18,7 @@ class Hpctoolkit(AutotoolsPackage):
     git      = "https://github.com/HPCToolkit/hpctoolkit.git"
 
     version('develop', branch='master')
+    version('gpu', branch='master-gpu')
```

Comment:
Note that we support `master` as a special, ahead-of-everything-except-develop version now.  `gpu` is not special and will compare as earlier than your other yyyy.mm.dd versions.  You could call these versions `master` and `master-gpu`, which might be more understandable to HPCToolkit users.

Reply:
I didn't realize that `master` had been added as a special name (along
with head and trunk, I see).  I'll change develop to master.

As for `gpu`, I'll check with @jmellorcrummey.  But I'm inclined to
keep it as gpu.  The name of the branch has already changed a couple
times.  `master-gpu` just means the merge of master and gpu, before
it's ready to be merged into master (at which time, we can drop the gpu
branch).